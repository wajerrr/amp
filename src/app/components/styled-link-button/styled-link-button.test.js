import React from 'react';
import renderer from 'react-test-renderer';
import StyledLinkButton from './styled-link-button';

describe('StyledLinkButton', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(
        <StyledLinkButton href="www.url.com">test button</StyledLinkButton>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
