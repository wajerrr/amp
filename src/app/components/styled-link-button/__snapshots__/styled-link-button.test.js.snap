// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledLinkButton should match snapshot 1`] = `
<a
  className="css-856nem-StyledLink e1ob2h4l0"
  href="www.url.com"
  role="button"
>
  test button
</a>
`;
